src/components/Article.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => (
+  <React.Fragment>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </React.Fragment>
```

Comment:
You can use shorter version of fragment, too
```suggestion
  <>
    <h1>{title}</h1>
    <span>{date}</span>
    <p>{text}</p>
  </>
```